service/storage_proxy.hh

```diff
@@ -388,6 +393,9 @@ private:
     void maybe_update_view_backlog_of(gms::inet_address, std::optional<db::view::update_backlog>);
 
     db::view::update_backlog get_backlog_of(gms::inet_address) const;
+
+    template<typename Range>
+    future<> mutate_counters(Range&& mutations, db::consistency_level cl, tracing::trace_state_ptr tr_state, service_permit permit, clock_type::time_point timeout);
```

Comment:
Why is it moved here?

Reply:
Move it to private section, just to discourage any (right-now non-existant) direct calls to this, as it is a sub-path to regular (augmented) paths.